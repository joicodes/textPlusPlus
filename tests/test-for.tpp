def void start(){
	int x;

	for(x = 0; x < 4; x = x + 1){
		write("Do re me");
	}
}