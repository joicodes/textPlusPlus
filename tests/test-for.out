Do re me
Do re me
Do re me
Do re me
 

